portfolio/src/main/webapp/script.js

```diff
@@ -83,18 +83,19 @@ function getNumComments(displayCount) {
       historyContainer.appendChild(createCommentDisplay(comment));
     });
     const moreButton = document.getElementsByName("more-comments")[0];
+    const countTextBox = document.getElementsByName("display-count")[0];
```

Comment:
should there be a length check before these or is there always guaranteed to be a defined 0th item?

Reply:
I think the elements should be guaranteed because they're created in the html. This is sort of the equivalent of doing document.getElementById() except idk why but when trying to get an element by name, the only method available for that returns the object in a list. So I guess should I do a length/null check every time I call getElementBy...()?

Reply:
Ah, I see. So long as they're literally in the HTML then I think you're fine :)